base-notebook/binder/postBuild

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 set -euo pipefail
 
-# serverextensions
-jupyter serverextension enable --sys-prefix --py nbserverproxy
 # nbextensions
 jupyter nbextension enable --sys-prefix --py widgetsnbextension
```

Comment:
can remove this since it happens automatically w/ ipywidgets installed via conda (https://ipywidgets.readthedocs.io/en/stable/user_install.html#with-conda)